cmd/google_cloud_ops_agent_engine/main.go

```diff
@@ -52,7 +50,14 @@ func main() {
 		log.Fatalf("The agent config file is not valid. Detailed error: %s", err)
 	}
 }
+
 func run() error {
+	if *healthChecks {
+		runHealthChecks()
```

Comment:
I have the concern that running with `--healthchecks` flag is going to run all of the Health Checks (Ports, Network, API), but the `Ports Check` is going to fail if the Ops Agent is running, causing confusion in the results. 

We can disable that `Ports Check` (not registering that check for this case) to avoid this.

Reply:
What if we just make the ports check short-circuit as a `PASS` if the Ops Agent (and subagents) are already running?

Reply:
Yeah, that's a good idea. I like it better than disabling the `Ports Check`. What is the best way to detect the Ops Agent is already running ?

Reply:
I guess there are at least two ways:

1. Check that the subagent services are running. If they're successfully running then necessarily they must be holding onto the ports, so it's a pass.
2. Check which processes (if any) own ports 20201 and 20202; if they match what we expect (e.g. `/opt/whatever/fluent-bit` and `/opt/whatever/otelopscol` for Linux, likewise for Windows) then the subagents are already running with those ports, so it's a pass.

I am probably missing some gotchas here so I'm curious to know what others think.

Reply:
2 might be somewhat problematic, since checking which process owns a port is often a privileged operation, and the health check invocation will likely be in user space… 1 could be sufficient.

Reply:
Yes, I'm also curious about what others think. This came up at the last discussion relating the `Ports Check Flakiness`. 

I would go with `1.` (but still not sure how to do it). `2.` (as mentioned by @igorpeshansky) would be problematic since probably running a `netstat` command or something similar is going to be platform/implementation dependent.

Reply:
On Linux: check the exit code of `systemctl is-active --quiet google-cloud-ops-agent-fluent-bit` (likewise for the metrics service)

On Windows: check the output of `sc query google-cloud-ops-agent-fluent-bit | findstr "STATE"` for `RUNNING` (likewise for the metrics service)